### PR TITLE
[Wallet] Add back trigger for enabling dev mode in setting menu

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

Add back trigger for enabling dev mode in setting menu. Clicks show no user feedback.

### Tested

In iOS simulator

### Backwards compatibility

Yes

### CODE REVIEW

**Comment by jmrossy** [comment]
TODO: add a toast message

_Edit_: Done.

File: packages/mobile/src/account/reducer.ts [hunk 1]
```diff
@@ -111,10 +112,14 @@ export const reducer = (
       }
     case Actions.DEV_MODE_TRIGGER_CLICKED:
       const newClickCount = (state.devModeClickCount + 1) % 6
+      const devModeActive = newClickCount >= 3
+      if (devModeActive) {
+        Logger.showMessage('Debug Mode Activated')
+      }
```
Comment (jeanregisser) [inline packages/mobile/src/account/reducer.ts:118]: nit: though this is harmless here, reducers shouldn't have any side effect.
This should be at the component level or saga.

**Review by jeanregisser — COMMENTED** [review]
Thanks! Just one minor thing.

**Review by i1skn — APPROVED** [review]
LGTM after @jeanregisser feedback is addressed :)

File: packages/mobile/src/account/reducer.ts (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/account/reducer.ts:118]: I considered that but that would require us to create a watcher for this event which duplicates the logic in this reducer. Just not worth it for 'purity' in this case I think

**Pull request merged**